internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -262,6 +263,12 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 		log.Error(err, "Error starting proxy.")
 		os.Exit(1)
 	}
+	// start the ansible-operator api server
+	err = apiserver.Run(done, apiserver.Options{})
```

Comment:
With the options included in the metric we could the listen address and port here instead, which I think would be more clear for future maintainers